inlang/source-code/paraglide/paraglide-js/src/cli/commands/init/command.test.ts

```diff
@@ -461,7 +468,9 @@ describe("maybeAddSherlock", () => {
 		])
 		await maybeAddSherlock({ project, logger, repo })
 		expect(consola.prompt).toHaveBeenCalledOnce()
-		expect(fs.writeFile).not.toHaveBeenCalled()
+		await expect(
+			async () => await fs.readFile("/.vscode/extensions.json", { encoding: "utf-8" })
+		).rejects.toThrow()
```

Comment:
was this change intentional?

Reply:
Yes, the caching was throwing off the "writeFile" count so I switched to a more reliable check. It might be even better to call `Sherlock.isAdopted` to check if it has **not** been adopted, but this works for now.